### PR TITLE
Add release notes 2.4, status name change on RestoreActivitiesPanel

### LINKED ISSUE #243: [Restore Snapshots/Indexes]
**Is your feature request related to a problem?**
In the 2.1 release, we introduced Snapshots Management with the ability to automatically create snapshots to backup user data from OpenSearch indexes. Today, users have to restore snapshots through the restore API. As a follow-up, we want to introduce restore functionality in Dashboards UI. Users looking for a guided experience will be able to restore their indexes from existing snapshots within OpenSearch Dashboards.

**What solution would you like?**
Add restore option on the Snapshots page on UI to allow users to select a snapshot to restore. 
Support restore parameters already available in the Restore API on UI. Specifically, we would like to allow restoring indices to a new index name. 
Support Restore Activity tab to monitor ongoing restore.

### PR DESCRIPTION
**Pull request by phestecAMZN**
### Description
Adds release notes for 2.4
Corrects minor issue caused by recent changes to stage names in RestoreActivitiesPanel

### Issues Resolved
#243 

### Check List
- [x ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Version 2.4.0.0 2022-11-04
```
Comment (bowenlan-amzn) [inline release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md:1]: file name should be -2.4.0.0

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Compatible with OpenSearch 2.4.0
+
+### Features
+* Add Snapshot Restore functionality and UI([#338](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/338))
+* Feature/create index([#289](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/289))
+* Feature/index operation ([#295](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/295))
+* Feature/index detail ([#312](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/312))
```
Comment (bowenlan-amzn) [inline release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md:9]: Not 2.4 changes

Comment (phestecAMZN) [reply release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md:9]: Thank you, done.

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md (hunk 1 above)
Comment (phestecAMZN) [reply release-notes/opensearch-index-management-dashboards-plugin.release-notes-4.0.0.0.md:1]: Thank you, done.

**Review by Angie-Zhang — APPROVED** [review]

**Pull request closed without merging**